ansible/roles/deploy-kubernetes/tasks/01_configure_systems.yaml

```diff
@@ -225,9 +235,7 @@
       state: absent
       
   - name: Add key from keyserver
-    apt_key:
```

Comment:
Why have you changed that? This module should work without any issue as intended.